src/patternfly/demos/Card/examples/CardView.md

```diff
@@ -284,3 +287,1028 @@ wrapperTag: div
+            {{#> description-list-text}}
+              {{#> l-flex}}
+                {{#> l-flex-item}}
+                  <i class="fas fa-exclamation-circle" aria-hidden="true"></i>
+                {{/l-flex-item}}
+                {{#> l-flex-item}}
+                  {{#> title titleType="h3" title--modifier="pf-m-md"}}
+                    Ready
+                  {{/title}}
+                {{/l-flex-item}}
+              {{/l-flex}}
+            {{/description-list-text}}
+          {{/description-list-term}}
+          {{#> description-list-description}}
+            {{#> description-list-text}}
+              <a href="#">Resource name that is long and can wrap</a>
+            {{/description-list-text}}
+            {{#> description-list-text}}
+              121 systems
+            {{/description-list-text}}
+          {{/description-list-description}}
+        {{/description-list-group}}
+      {{/description-list}}
+    {{/tab-content}}
+    {{#> tab-content tab-content--attribute=(concat 'id="' primary-detail-template--id '-tab2-panel"')}}
+      Panel 2
+    {{/tab-content}}
+    {{#> tab-content tab-content--attribute=(concat 'id="' primary-detail-template--id '-tab3-panel"')}}
+      Panel 3
+    {{/tab-content}}
+  {{/card-body}}
+{{/card}}
+```
+
+### Status card
+```hbs
+{{> cardview-demo--status-card card--id="status-card-default-example"}}
+```
+
+### Status card expanded notifications
+```hbs
+{{> cardview-demo--status-card card--id="status-card-expanded-example" cardview-demo--notication-drawer--IsOpen="true"}}
+```
+
+### Status card expanded with popover
+```hbs
+{{> cardview-demo--status-card card--id="status-card-expanded-with-popover-example" cardview-demo--popover--IsOpen="true" cardview-demo--notication-drawer--IsOpen="true"}}
+```
+
+### Utilization card 1
+```hbs
+{{#> gallery gallery--modifier="pf-m-gutter" gallery--attribute='style="--pf-l-gallery--GridTemplateColumns--min: 360px;"'}}
+  {{#> card card--id="utilization-card-1-example"}}
+    {{#> card-title card-title--attribute=(concat 'id="' card--id '-title1"')}}
+      {{#> title title--modifier="pf-m-lg" titleType="h2"}}
+        Top Utilized Clusters
+      {{/title}}
+    {{/card-title}}
+    {{#> card-body}}
+      {{#> l-flex l-flex--modifier="pf-m-column pf-m-space-items-xs"}}
+        {{#> l-flex-item}}
+          {{#> stack}}
+            {{#> stack-item}}
+              <b>Cluster-1204</b>
+            {{/stack-item}}
+            {{#> stack-item}}
+              <span>27.3 cores available</span>
+            {{/stack-item}}
+          {{/stack}}
+        {{/l-flex-item}}
+        {{#> l-flex-item l-flex-item--modifier="pf-m-spacer-md"}}
+          {{> cardview-demo--chart}}
+        {{/l-flex-item}}
+        {{#> l-flex-item}}
+          <a href="#">View details</a>
+        {{/l-flex-item}}
+      {{/l-flex}}
+    {{/card-body}}
+    {{#> card-body}}
+      {{#> l-flex l-flex--modifier="pf-m-column pf-m-space-items-none"}}
```

Comment:
In the block above this, looks like the same layout and you use `space-items-xs`. Should this be updated to match?